ttl/hbp_macaca_fascicularis_brain_ontology.ttl

```diff
@@ -0,0 +1,657 @@
+
+                 nsu:synonym "V2"@en .
+
+
+MFBA:00007 rdf:type owl:Class ;
+
+                 rdfs:label "area TE, anterior-dorsal part"@en ;
+
+                 nsu:synonym "TEad"@en .
+
+
+MFBA:00008 rdf:type owl:Class ;
+
+                 rdfs:label "visual area 4"@en ;
+
+                 nsu:synonym "V4"@en .
+
+
+MFBA:00009 rdf:type owl:Class ;
+
+                 rdfs:label "area TEO"@en ;
+
+                 nsu:synonym "TEO"@en .
+
+
+MFBA:00010 rdf:type owl:Class ;
+
+                 rdfs:label "area TE, posterior-dorsal part"@en ;
+
+                 nsu:synonym "TEpd"@en .
+
+
+MFBA:00011 rdf:type owl:Class ;
+
+                 rdfs:label "subiculum"@en ;
+
+                 nsu:synonym "SUB"@en .
+
+
+MFBA:00012 rdf:type owl:Class ;
+
+                 rdfs:label "temporal pole"@en ;
+
+                 nsu:synonym "POLE"@en .
+
+
+MFBA:00013 rdf:type owl:Class ;
+
+                 rdfs:label "superior temporal polysensory, rostral part"@en ;
+
+                 nsu:synonym "STPr"@en .
+
+
+MFBA:00014 rdf:type owl:Class ;
+
+                 rdfs:label "visual area 1"@en ;
+
+                 nsu:synonym "V1"@en .
+
+
+MFBA:00015 rdf:type owl:Class ;
+
+                 rdfs:label "area IPa"@en ;
+
+                 nsu:synonym "IPa"@en .
+
+
+MFBA:00016 rdf:type owl:Class ;
+
+                 rdfs:label "area PGa"@en ;
+
+                 nsu:synonym "PGa"@en .
+
+
+MFBA:00017 rdf:type owl:Class ;
+
+                 rdfs:label "parabelt, rostral part"@en ;
+
+                 nsu:synonym "PBr"@en .
+
+
+MFBA:00018 rdf:type owl:Class ;
+
+                 rdfs:label "insula"@en ;
+
+                 nsu:synonym "INS"@en .
+
+
+MFBA:00019 rdf:type owl:Class ;
+
+                 rdfs:label "superior temporal polysensory, intermediate part"@en ;
+
+                 nsu:synonym "STPi"@en .
+
+
+MFBA:00020 rdf:type owl:Class ;
+
+                 rdfs:label "parainsula"@en ;
+
+                 nsu:synonym "Pi"@en .
+
+
+MFBA:00021 rdf:type owl:Class ;
+
+                 rdfs:label "prostriata"@en ;
+
+                 nsu:synonym "ProSt"@en .
+
+
+MFBA:00022 rdf:type owl:Class ;
+
+                 rdfs:label "fundus of superior temporal area"@en ;
+
+                 nsu:synonym "FST"@en .
+
+
+MFBA:00023 rdf:type owl:Class ;
+
+                 rdfs:label "area TEO medial part"@en ;
+
+                 nsu:synonym "TEOm"@en .
+
+
+MFBA:00024 rdf:type owl:Class ;
+
+                 rdfs:label "middle temporal area"@en ;
+
+                 nsu:synonym "MT"@en .
+
+
+MFBA:00025 rdf:type owl:Class ;
+
+                 rdfs:label "medial belt"@en ;
+
+                 nsu:synonym "MB"@en .
+
+
+MFBA:00026 rdf:type owl:Class ;
+
+                 rdfs:label "piriform"@en ;
+
+                 nsu:synonym "PIR"@en .
+
+
+MFBA:00027 rdf:type owl:Class ;
+
+                 rdfs:label "orbital proisocortex"@en ;
+
+                 nsu:synonym "OPRO"@en .
+
+
+MFBA:00028 rdf:type owl:Class ;
```

Comment:
Macaca fascicularis does not have a hierarchy, we could add Brain as parent of all the regions.